build.sbt

```diff
@@ -3,17 +3,14 @@ import Dependencies.*
 ThisBuild / scalaVersion := "2.12.15"
 ThisBuild / version := "1.0-SNAPSHOT"
 ThisBuild / organization := "com.gradle"
-ThisBuild / organizationName := "gradle"
+ThisBuild / organizationName := "Gradle Inc."
 
 sbtPlugin := true
-publishMavenStyle := true
-resolvers += Resolver.mavenLocal
 
 Global / develocityConfiguration :=
   DevelocityConfiguration(
     server = Server(
-//      url = Some(url("https://ge.solutions-team.gradle.com"))
-        url = Some(url("https://ge-helm-cluster-unstable.grdev.net"))
+      url = Some(url("https://ge.solutions-team.gradle.com"))
     ),
     buildScan = BuildScan(
       tags = Set(),
```

Comment:
Why keeping this?

Reply:
Because we want to publish scans of sbt-ccud runs to solutions dv instance (since it's on 2024.1 already).

Reply:
Oh 🤦 . Yeah, I'll definitely remove that. The plan is to add the sbt ccud plugin to itself, but it needs to be released first :)

Reply:
It doesn't actually: it's possible to use the plugin directly in its build. If this is the approach that you would prefer, I can send a patch to add that.

Reply:
Sounds good, what do you think @runningcode?

Reply:
I'm confused what this thread is about since it seems there are two topics being discussed but if you are asking me if we should add the plugin to this build just like we do for all the other CCUD plugins/extensions then the answer is yes.